dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -29,13 +30,14 @@ class EvaluateRubricJob < ApplicationJob
     'allthethings' => {
       'CSD U3 Sprites scene challenge_allthethings' => 'allthethings-L48',
     },
+    # TODO: re-enable these once rubrics have been added via levelbuilder
     'interactive-games-animations-2023' => {
-      'CSD U3 Sprites scene challenge_2023' => 'csd3-2023-L11',
-      'CSD web project animated review_2023' => 'csd3-2023-L14',
+      # 'CSD U3 Sprites scene challenge_2023' => 'csd3-2023-L11',
+      # 'CSD web project animated review_2023' => 'csd3-2023-L14',
       'CSD U3 Interactive Card Final_2023' => 'csd3-2023-L18',
-      'CSD games sidescroll review_2023' => 'csd3-2023-L21',
-      'CSD U3 collisions flyman bounceOff_2023' => 'csd3-2023-L24',
-      'CSD games project review_2023' => 'csd3-2023-L28',
+      # 'CSD games sidescroll review_2023' => 'csd3-2023-L21',
+      # 'CSD U3 collisions flyman bounceOff_2023' => 'csd3-2023-L24',
+      # 'CSD games project review_2023' => 'csd3-2023-L28',
```

Comment:
each of these lines that's been disabled here is a "problem lesson" caught by `validate_learning_goals`. These lessons do not have corresponding rubrics in levelbuilder. the work to fix this is tracked in https://codedotorg.atlassian.net/browse/AITT-496 .